projects/framework-nuke/extensions/tool-configs/nuke-image-loader.yaml

```diff
@@ -0,0 +1,35 @@
+type: tool_config
+name: nuke-image-loader
+config_type: loader
+compatible:
+  entity_types:
+    - FileComponent
+  supported_file_extensions:
+    - ".png"
+    - ".jpg"
+    - ".jpeg"
+    - ".exr"
+    - ".tif"
+    - ".tiff"
+    - ".tga"
+    - ".bmp"
+    - ".hdr"
+    - ".dpx"
+    - ".cin"
+    - ".psd"
+    - ".tx"
+
+engine:
+  - type: plugin
+    tags:
+      - context
+    plugin: resolve_entity_path
+    ui: show_component
+
+  - type: plugin
+    tags:
+      - loader
+    options:
+      name: "Nuke image loader"
```

Comment:
Why are we passing name inside options? Is this really needed?
Can't you put logic inside the plugin or the widget to get the name instead?